### PR TITLE
Reuse the File, Config field, and file-related validators from flask-marshmallow

### PR DESCRIPTION
**Pull request by uncle-lv**
This pr is for #540 .

### CODE REVIEW

**Comment by uncle-lv** [comment]
I don't know what the file [min-versions.in](https://github.com/apiflask/apiflask/blob/f62ec12b8214b23495dc7cba56ee410a263753c9/requirements/min-versions.in#L3) does. But if I pin flask-marshmallow to 1.0.0 version, I get an error.
```
ERROR! requirements/min-versions.txt was not regenerated after changes in requirements/min-versions.in.
Expecting: # SHA1:3725672cc8aa4cdcab4174d189f19e8bb3ac2b62
Found:     # SHA1:a74d3671a740ec5c076a37aaaa3409fd78f9bde2
```

**Comment by greyli** [comment]
> I don't know what the file [min-versions.in](https://github.com/apiflask/apiflask/blob/f62ec12b8214b23495dc7cba56ee410a263753c9/requirements/min-versions.in#L3) does. But if I pin flask-marshmallow to 1.0.0 version, I get an error.
> 
> ```
> ERROR! requirements/min-versions.txt was not regenerated after changes in requirements/min-versions.in.
> Expecting: # SHA1:3725672cc8aa4cdcab4174d189f19e8bb3ac2b62
> Found:     # SHA1:a74d3671a740ec5c076a37aaaa3409fd78f9bde2
> ```

The `min-versions.in` file (input) contains the minimal supported version of the dependencies. You could update the version in it, then use `pip-compile-multi` or `pip-compile requirements/min-versions.in` command to generate the `min-versions.txt` file (output, lock file).

Ref:

* https://github.com/jazzband/pip-tools
* https://github.com/peterdemin/pip-compile-multi

File: tests/test_helpers.py [hunk 1]
```diff
@@ -1,103 +0,0 @@
...
-    assert rv.json['last'].endswith('/pets?page=100&per_page=20')
-    assert rv.json['next'].endswith('/pets?page=2&per_page=20')
-    assert rv.json['prev'].endswith('')
-    assert rv.json['current'].endswith('/pets?page=1&per_page=20')
-    assert rv.json['per_page'] == 20
-    assert rv.json['pages'] == 100
-    assert 'has_next' not in rv.json
-    assert 'next_num' not in rv.json
-    assert 'has_prev' not in rv.json
-    assert 'prev_num' not in rv.json
```
Comment (greyli) [inline tests/test_helpers.py:64]: This file should be kept as it contains the tests for existing helper functions.

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ## Version 2.1.1
 
 Released: -
+ - Reuse the File, Config field, and file-related validators from flask-marshmallow ([issue #540][issue_540]).
+
+ [issue_540]: https://github.com/apiflask/apiflask/issues/540
```
Comment (greyli) [inline CHANGES.md:6]: There are some unnecessary whitespaces here.

```suggestion
- Reuse the File, Config field, and file-related validators from flask-marshmallow ([issue #540][issue_540]).

[issue_540]: https://github.com/apiflask/apiflask/issues/540
```

**Review by greyli — CHANGES_REQUESTED** [review]

**Review by greyli — APPROVED** [review]

**Comment by greyli** [comment]
Thanks!

**Pull request merged**